hoodie-client/src/test/java/HoodieClientExample.java

```diff
@@ -54,16 +86,15 @@ public static void main(String[] args) throws Exception {
 
         // generate some records to be loaded in.
         HoodieWriteConfig cfg =
-            HoodieWriteConfig.newBuilder().withPath(tablePath)
+            HoodieWriteConfig.newBuilder().withPath(inputTablePath)
                 .withSchema(HoodieTestDataGenerator.TRIP_EXAMPLE_SCHEMA).withParallelism(2, 2)
-                .forTable("sample-table").withIndexConfig(
+                .forTable(DEFAULT_TABLE_NAME).withIndexConfig(
```

Comment:
This should be - inputTableName. Will wait for other review comments and add a fix for this.